cves/CVE-2002-0661.yml

```diff
@@ -118,10 +131,14 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
-    - commit:
-      note:
+    - commit: 238a9d6e51ba186e6903272c16a6ec695e3654d6
+      note: |
+        Much development occurred on mod_negotiation.c, the primary file containing 
+        the vulnerability, but little was relevant to the vulnerability as the file 
+        is quite large. This is the only commit which works with this area of the code. 
+        It adds a comment which illustrates that the author considered path traversal 
+        and the current way it is handled but did not consider that checking for '/' 
+        would not be sufficient.
```

Comment:
Potentially change this last line to "would be insufficient."

Reply:
yeah that's a bit more concise